loader_hub/dadjokes/base.py

```diff
@@ -0,0 +1,32 @@
+"""icanhazdadjoke based dad-joke generator."""
```

Comment:
Can we name this `dad_jokes` instead? Also, you'll need to add it to the `library.json` file for it to be picked up by `gpt_index.download_loader` or the website.